src/CommonTestBase.sol

```diff
@@ -42,36 +42,42 @@ contract CommonTestBase is Test {
       if (asset == AaveV2EthereumAssets.GUSD_UNDERLYING) {
         vm.startPrank(0x22FFDA6813f4F34C520bf36E5Ea01167bC9DF159);
```

Comment:
I would just use `vm.prank(...)` instead of start/stop here.

Reply:
Agreed